modules/update-service-overview.adoc

```diff
@@ -9,17 +9,17 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="update-service-overview_{context}"]
-= About the {product-title} update service
+= About the {product-title} Update Service
 
-The {product-title} update service is the hosted service that provides over-the-air
+The {product-title} Update Service is the service that provides over-the-air
```

Comment:
Can we introduce the OSUS acronym here (or somewhere) for readability and to start socializing the acronym.

Reply:
@wking, do you know if this is an acceptable acronym to socialize?

Reply:
I don't like socializing acronyms, but don't block on me ;)

Reply:
@kalexand-rh I assume we have an established "standard" in documents about whether and/or when we use acronyms? Assuming so I'm not suggesting we break new ground so just roll with the standard.

Reply:
@jottofar, I've seen hard rules for only product-level initializations. OCP is forbidden, but OLM is fine. CVO is used throughout the docs, so I see no _problem_ with using OSUS. It's a judgement call, but I'm not convinced that it's mine to make.

Reply:
If customers are going to be told that this is OSUS, then I think we should go ahead and use the acronym. It's apparently confusing that customer-facing folks refer to IPI and UPI installation methods but docs don't.

Reply:
Going back and looking at this again I see I have it wrong. I was originally referring to OpenShift Container Platform Update Service. But OSUS, which is used by the team, is the acronym for OpenShift Update Service. That name was chosen, at least partly, because OpenShift Container Platform Update Service is too much of a mouthful. I think the action here is to change all occurrences of `OpenShift Container Platform Update Service` to `OpenShift Update Service`. And the latter is short enough to not worry about referring to the acronym. I don't think we want to have `OpenShift Container Platform OpenShift Update Service` unless we're breaking some consistency rule.

Reply:
Sorry, just remembered {product-title}. So I'm sure we need to keep {product-title} which means change would be to `OpenShift Container Platform OpenShift Update Service`. As I said, that really is a mouthful and it's used throughout so using OSUS, i.e. `OpenShift Container Platform OSUS`, seems reasonable to me.

Reply:
I've changed this from  `OpenShift Container Platform Update Service` to `OpenShift Update Service` throughout.